src/store/api/leaderboard/transformer.js

```diff
@@ -0,0 +1,45 @@
+import { orderBy } from "lodash";
```

Comment:
import this as `import { default as orderBy } from "lodash/orderBy";`

Reply:
Done